src/probe_modules/module_bacnet.c

```diff
@@ -73,7 +73,8 @@ int bacnet_init_perthread(void *buf, macaddr_t *src,
 }
 
 int bacnet_make_packet(void *buf, ipaddr_n_t src_ip, ipaddr_n_t dst_ip,
-				uint32_t *validation, int probe_num, UNUSED void *arg)
+				uint32_t *validation, int probe_num, UNUSED void *arg,
+				UNUSED int *len)
```

Comment:
`buf_len` is okay, but it did take me a moment to realize that it was a pointer, because I'm used to lengths being not pointers-to-length.